settings.gradle

```diff
@@ -1 +1,3 @@
-include ':AndroidStealth', ':libraries:FileChooser', ':libraries:droidmorphing-lib', ':libraries:DirectoryChooser'
+include ':AndroidStealth', ':libraries:FileChooser', ':libraries:DirectoryChooser', ':droidmorphing-lib'
+project(':droidmorphing-lib').projectDir = new File('libraries/droid-morph/droidmorphing-lib')
```

Comment:
This is the magic trick required with submodules. Because of this, it doesn't try to 'build' the `droid-morph` folder, which cannot be built.